src/metakb/schemas/annotation.py

```diff
@@ -4,7 +4,7 @@
 from enum import Enum
 from typing import Literal
 
-from ga4gh.core import core_models
+from ga4gh.core.entity_models import IRI, Coding, _DomainEntity, _Entity
```

Comment:
not actionable here, but if we're importing "private" classes like `_DomainEntity` here, maybe they shouldn't be denoted as "private"

Reply:
Ya I was thinking the same. I thought about inheriting from ABC and removing underscore but haven't made an issue